[2024-01-04T20:26:00.459+0000] {processor.py:161} INFO - Started process (PID=3785) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:26:00.461+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:26:00.462+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:26:00.462+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:26:00.476+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:26:00.475+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:26:00.478+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:26:00.520+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.064 seconds
[2024-01-04T20:26:30.934+0000] {processor.py:161} INFO - Started process (PID=3843) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:26:30.935+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:26:30.937+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:26:30.936+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:26:30.954+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:26:30.953+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:26:30.955+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:26:30.977+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.047 seconds
[2024-01-04T20:27:01.885+0000] {processor.py:161} INFO - Started process (PID=3901) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:27:01.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:27:01.888+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:27:01.888+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:27:01.910+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:27:01.909+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:27:01.911+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:27:01.937+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.057 seconds
[2024-01-04T20:27:32.119+0000] {processor.py:161} INFO - Started process (PID=3959) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:27:32.121+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:27:32.122+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:27:32.122+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:27:32.141+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:27:32.140+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:27:32.141+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:27:32.168+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.052 seconds
[2024-01-04T20:28:02.618+0000] {processor.py:161} INFO - Started process (PID=4017) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:28:02.620+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:28:02.622+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:28:02.622+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:28:02.655+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:28:02.654+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:28:02.656+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:28:02.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.072 seconds
[2024-01-04T20:28:32.819+0000] {processor.py:161} INFO - Started process (PID=4075) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:28:32.821+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:28:32.822+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:28:32.822+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:28:32.842+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:28:32.841+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:28:32.842+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:28:32.872+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.057 seconds
[2024-01-04T20:29:03.254+0000] {processor.py:161} INFO - Started process (PID=4133) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:29:03.256+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:29:03.258+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:29:03.257+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:29:03.288+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:29:03.287+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:29:03.289+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:29:03.322+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.073 seconds
[2024-01-04T20:29:33.591+0000] {processor.py:161} INFO - Started process (PID=4191) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:29:33.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:29:33.593+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:29:33.593+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:29:33.612+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:29:33.611+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:29:33.613+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:29:33.641+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.055 seconds
[2024-01-04T20:30:03.933+0000] {processor.py:161} INFO - Started process (PID=4249) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:30:03.935+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:30:03.937+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:30:03.937+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:30:03.966+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:30:03.965+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:30:03.967+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:30:03.994+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.065 seconds
[2024-01-04T20:30:34.143+0000] {processor.py:161} INFO - Started process (PID=4307) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:30:34.144+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:30:34.146+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:30:34.145+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:30:34.164+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:30:34.163+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:30:34.165+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:30:34.193+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.054 seconds
[2024-01-04T20:31:05.055+0000] {processor.py:161} INFO - Started process (PID=4365) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:31:05.058+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:31:05.061+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:31:05.060+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:31:05.092+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:31:05.091+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:31:05.092+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:31:05.124+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.082 seconds
[2024-01-04T20:31:35.868+0000] {processor.py:161} INFO - Started process (PID=4423) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:31:35.870+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:31:35.872+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:31:35.871+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:31:35.892+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:31:35.892+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:31:35.893+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:31:35.921+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.057 seconds
[2024-01-04T20:32:06.046+0000] {processor.py:161} INFO - Started process (PID=4481) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:32:06.048+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:32:06.050+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:32:06.049+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:32:06.069+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:32:06.067+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:32:06.070+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:32:06.094+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.052 seconds
[2024-01-04T20:32:36.140+0000] {processor.py:161} INFO - Started process (PID=4539) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:32:36.142+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:32:36.144+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:32:36.144+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:32:36.164+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:32:36.163+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:32:36.165+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:32:36.195+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.059 seconds
[2024-01-04T20:33:06.721+0000] {processor.py:161} INFO - Started process (PID=4596) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:33:06.723+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:33:06.724+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:33:06.724+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:33:06.752+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:33:06.751+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:33:06.753+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:33:06.781+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.063 seconds
[2024-01-04T20:33:36.954+0000] {processor.py:161} INFO - Started process (PID=4654) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:33:36.955+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:33:36.957+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:33:36.956+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:33:36.972+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:33:36.971+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:33:36.973+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:33:37.000+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.049 seconds
[2024-01-04T20:34:07.142+0000] {processor.py:161} INFO - Started process (PID=4712) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:34:07.144+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:34:07.145+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:34:07.145+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:34:07.167+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:34:07.166+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:34:07.168+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:34:07.198+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.060 seconds
[2024-01-04T20:34:37.524+0000] {processor.py:161} INFO - Started process (PID=4770) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:34:37.526+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:34:37.527+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:34:37.527+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:34:37.552+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:34:37.551+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:34:37.553+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:34:37.584+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.065 seconds
[2024-01-04T20:35:08.139+0000] {processor.py:161} INFO - Started process (PID=4828) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:35:08.140+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:35:08.142+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:35:08.142+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:35:08.168+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:35:08.167+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:35:08.169+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:35:08.203+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.070 seconds
[2024-01-04T20:35:38.822+0000] {processor.py:161} INFO - Started process (PID=4886) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:35:38.823+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:35:38.825+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:35:38.825+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:35:38.848+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:35:38.847+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:35:38.849+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:35:38.880+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.062 seconds
[2024-01-04T20:36:09.597+0000] {processor.py:161} INFO - Started process (PID=4944) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:36:09.598+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:36:09.600+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:36:09.600+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:36:09.624+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:36:09.623+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:36:09.626+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:36:09.652+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.058 seconds
[2024-01-04T20:36:40.225+0000] {processor.py:161} INFO - Started process (PID=5002) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:36:40.226+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:36:40.227+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:36:40.227+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:36:40.243+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:36:40.242+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:36:40.244+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:36:40.274+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.053 seconds
[2024-01-04T20:37:11.005+0000] {processor.py:161} INFO - Started process (PID=5060) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:37:11.010+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:37:11.012+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:37:11.011+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:37:11.038+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:37:11.036+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:37:11.038+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:37:11.072+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.071 seconds
[2024-01-04T20:37:41.554+0000] {processor.py:161} INFO - Started process (PID=5118) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:37:41.560+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:37:41.562+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:37:41.561+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:37:41.585+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:37:41.583+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:37:41.586+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:37:41.635+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.086 seconds
[2024-01-04T20:38:12.066+0000] {processor.py:161} INFO - Started process (PID=5176) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:38:12.069+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:38:12.071+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:38:12.071+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:38:12.096+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:38:12.095+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:38:12.097+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:38:12.128+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.066 seconds
[2024-01-04T20:38:42.470+0000] {processor.py:161} INFO - Started process (PID=5234) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:38:42.472+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:38:42.473+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:38:42.473+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:38:42.490+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:38:42.490+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:38:42.491+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:38:42.525+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.058 seconds
[2024-01-04T20:39:13.145+0000] {processor.py:161} INFO - Started process (PID=5292) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:39:13.147+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:39:13.149+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:39:13.149+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:39:13.170+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:39:13.169+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:39:13.171+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:39:13.198+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.058 seconds
[2024-01-04T20:39:43.953+0000] {processor.py:161} INFO - Started process (PID=5351) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:39:43.964+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:39:43.966+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:39:43.965+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:39:43.999+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:39:43.997+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:39:44.001+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:39:44.044+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.100 seconds
[2024-01-04T20:40:14.872+0000] {processor.py:161} INFO - Started process (PID=5409) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:40:14.874+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:40:14.876+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:40:14.875+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:40:14.901+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:40:14.900+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:40:14.902+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:40:14.948+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.081 seconds
[2024-01-04T20:40:45.678+0000] {processor.py:161} INFO - Started process (PID=5467) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:40:45.681+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:40:45.684+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:40:45.683+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:40:45.712+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:40:45.711+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:40:45.713+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:40:45.758+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.088 seconds
[2024-01-04T20:41:15.959+0000] {processor.py:161} INFO - Started process (PID=5525) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:41:15.962+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:41:15.963+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:41:15.963+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:41:15.996+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:41:15.995+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:41:15.997+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:41:16.030+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.075 seconds
[2024-01-04T20:41:46.190+0000] {processor.py:161} INFO - Started process (PID=5583) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:41:46.192+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:41:46.193+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:41:46.193+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:41:46.209+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:41:46.208+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:41:46.210+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:41:46.238+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.051 seconds
[2024-01-04T20:42:16.343+0000] {processor.py:161} INFO - Started process (PID=5641) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:42:16.346+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:42:16.347+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:42:16.347+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:42:16.371+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:42:16.370+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:42:16.372+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:42:16.408+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.068 seconds
[2024-01-04T20:42:47.027+0000] {processor.py:161} INFO - Started process (PID=5699) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:42:47.030+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:42:47.035+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:42:47.035+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:42:47.056+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:42:47.055+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:42:47.056+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:42:47.098+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.078 seconds
[2024-01-04T20:43:17.496+0000] {processor.py:161} INFO - Started process (PID=5757) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:17.499+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:43:17.500+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:17.500+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:17.530+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:17.528+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:43:17.531+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:17.568+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.080 seconds
[2024-01-04T20:43:48.506+0000] {processor.py:161} INFO - Started process (PID=5815) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:48.510+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:43:48.514+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:48.512+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:48.589+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:48.588+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/dataset_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dataset_dag.py", line 13
    )
    ^
SyntaxError: invalid syntax
[2024-01-04T20:43:48.590+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:48.665+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.170 seconds
[2024-01-04T20:43:56.575+0000] {processor.py:161} INFO - Started process (PID=5833) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:56.578+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:43:56.579+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:56.579+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:56.647+0000] {processor.py:840} INFO - DAG(s) 'dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:43:56.983+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:56.979+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:dag_dataset
[2024-01-04T20:43:57.017+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:57.016+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:dag_dataset
[2024-01-04T20:43:57.041+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:57.039+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:dag_dataset
[2024-01-04T20:43:57.043+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:57.043+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:43:57.087+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:57.086+0000] {dag.py:3055} INFO - Creating ORM DAG for dag_dataset
[2024-01-04T20:43:57.130+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:43:57.130+0000] {dag.py:3820} INFO - Setting next_dagrun for dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:43:57.263+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.699 seconds
[2024-01-04T20:44:27.435+0000] {processor.py:161} INFO - Started process (PID=5891) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:44:27.436+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:44:27.438+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:44:27.437+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:44:27.473+0000] {processor.py:840} INFO - DAG(s) 'dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:44:27.500+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:44:27.499+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:44:27.538+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:44:27.537+0000] {dag.py:3820} INFO - Setting next_dagrun for dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:44:27.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.147 seconds
[2024-01-04T20:44:57.911+0000] {processor.py:161} INFO - Started process (PID=5947) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:44:57.914+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:44:57.916+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:44:57.915+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:44:57.971+0000] {processor.py:840} INFO - DAG(s) 'dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:44:58.002+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:44:58.002+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:44:58.046+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:44:58.045+0000] {dag.py:3820} INFO - Setting next_dagrun for dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:44:58.101+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.197 seconds
[2024-01-04T20:45:28.245+0000] {processor.py:161} INFO - Started process (PID=6005) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:45:28.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:45:28.257+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:45:28.256+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:45:28.330+0000] {processor.py:840} INFO - DAG(s) 'dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:45:28.414+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:45:28.413+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:45:28.510+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:45:28.510+0000] {dag.py:3820} INFO - Setting next_dagrun for dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:45:28.672+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.445 seconds
[2024-01-04T20:45:59.132+0000] {processor.py:161} INFO - Started process (PID=6063) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:45:59.133+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:45:59.134+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:45:59.134+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:45:59.161+0000] {processor.py:840} INFO - DAG(s) 'dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:45:59.189+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:45:59.188+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:45:59.219+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:45:59.219+0000] {dag.py:3820} INFO - Setting next_dagrun for dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:45:59.263+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.135 seconds
[2024-01-04T20:46:02.734+0000] {processor.py:161} INFO - Started process (PID=6072) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:02.735+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:46:02.737+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.736+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:02.776+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:02.923+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.922+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:write_dag_dataset
[2024-01-04T20:46:02.942+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.941+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:write_dag_dataset
[2024-01-04T20:46:02.952+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.951+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:write_dag_dataset
[2024-01-04T20:46:02.952+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.952+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:46:02.965+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.965+0000] {dag.py:3055} INFO - Creating ORM DAG for write_dag_dataset
[2024-01-04T20:46:02.978+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:02.978+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:46:03.018+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.289 seconds
[2024-01-04T20:46:33.176+0000] {processor.py:161} INFO - Started process (PID=6130) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:33.177+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:46:33.178+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:33.178+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:33.206+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:33.228+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:33.228+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:46:33.259+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:33.258+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:46:33.302+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.130 seconds
[2024-01-04T20:46:36.702+0000] {processor.py:161} INFO - Started process (PID=6137) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:36.704+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:46:36.707+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:36.706+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:36.776+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:36.803+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:36.803+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:46:36.847+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:36.847+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:46:36.925+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.226 seconds
[2024-01-04T20:46:47.620+0000] {processor.py:161} INFO - Started process (PID=6159) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:47.623+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:46:47.627+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:47.625+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:47.712+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:46:47.770+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:47.770+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:46:47.848+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:46:47.848+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:46:47.984+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.371 seconds
[2024-01-04T20:47:18.167+0000] {processor.py:161} INFO - Started process (PID=6218) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:47:18.168+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:47:18.170+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:47:18.169+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:47:18.196+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:47:18.217+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:47:18.216+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:47:18.244+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:47:18.244+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:47:18.282+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.119 seconds
[2024-01-04T20:47:49.611+0000] {processor.py:161} INFO - Started process (PID=6276) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:47:49.613+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:47:49.616+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:47:49.615+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:47:49.662+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:47:49.714+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:47:49.713+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:47:49.773+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:47:49.772+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:47:49.836+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.232 seconds
[2024-01-04T20:48:20.029+0000] {processor.py:161} INFO - Started process (PID=6334) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:48:20.031+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:48:20.032+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:48:20.032+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:48:20.076+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:48:20.101+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:48:20.101+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:48:20.130+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:48:20.130+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:48:20.165+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.141 seconds
[2024-01-04T20:48:50.274+0000] {processor.py:161} INFO - Started process (PID=6386) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:48:50.276+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:48:50.277+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:48:50.277+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:48:50.307+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:48:50.337+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:48:50.337+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:48:50.375+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:48:50.374+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:48:50.463+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.194 seconds
[2024-01-04T20:49:21.183+0000] {processor.py:161} INFO - Started process (PID=6444) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:49:21.185+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:49:21.187+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:49:21.186+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:49:21.224+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:49:21.256+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:49:21.256+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:49:21.299+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:49:21.298+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:49:21.345+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.169 seconds
[2024-01-04T20:49:51.445+0000] {processor.py:161} INFO - Started process (PID=6502) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:49:51.447+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:49:51.448+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:49:51.448+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:49:51.479+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:49:51.513+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:49:51.513+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:49:51.566+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:49:51.566+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:49:51.621+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.181 seconds
[2024-01-04T20:50:22.501+0000] {processor.py:161} INFO - Started process (PID=6560) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:50:22.503+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:50:22.505+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:50:22.505+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:50:22.555+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:50:22.618+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:50:22.618+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:50:22.696+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:50:22.696+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:50:23.020+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.528 seconds
[2024-01-04T20:50:53.732+0000] {processor.py:161} INFO - Started process (PID=6618) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:50:53.734+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:50:53.735+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:50:53.735+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:50:53.764+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:50:53.801+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:50:53.801+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:50:53.853+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:50:53.853+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:50:53.919+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.194 seconds
[2024-01-04T20:51:24.204+0000] {processor.py:161} INFO - Started process (PID=6678) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:51:24.206+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:51:24.208+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:51:24.208+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:51:24.253+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:51:24.304+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:51:24.303+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:51:24.354+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:51:24.354+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:51:24.402+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.204 seconds
[2024-01-04T20:51:54.876+0000] {processor.py:161} INFO - Started process (PID=6737) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:51:54.883+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:51:54.893+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:51:54.891+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:51:54.965+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:51:55.013+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:51:55.012+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:51:55.068+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:51:55.067+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:51:55.134+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.267 seconds
[2024-01-04T20:52:25.997+0000] {processor.py:161} INFO - Started process (PID=6796) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:52:26.013+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:52:26.068+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:52:26.068+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:52:26.129+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:52:26.164+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:52:26.163+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:52:26.244+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:52:26.244+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:52:26.330+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.339 seconds
[2024-01-04T20:52:56.895+0000] {processor.py:161} INFO - Started process (PID=6856) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:52:56.897+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:52:56.899+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:52:56.899+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:52:56.984+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:52:57.056+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:52:57.055+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:52:57.184+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:52:57.183+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-03T00:00:00+00:00, run_after=2024-01-04T00:00:00+00:00
[2024-01-04T20:52:57.258+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.377 seconds
[2024-01-04T20:53:27.677+0000] {processor.py:161} INFO - Started process (PID=6915) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:53:27.679+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:53:27.685+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:53:27.683+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:53:27.952+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:53:28.250+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:53:28.249+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:53:28.485+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:53:28.485+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:53:28.620+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.953 seconds
[2024-01-04T20:53:59.308+0000] {processor.py:161} INFO - Started process (PID=6974) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:53:59.311+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:53:59.312+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:53:59.312+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:53:59.353+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:53:59.383+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:53:59.382+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:53:59.418+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:53:59.418+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:53:59.576+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.272 seconds
[2024-01-04T20:54:29.641+0000] {processor.py:161} INFO - Started process (PID=7033) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:54:29.643+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:54:29.644+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:54:29.644+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:54:29.686+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:54:29.724+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:54:29.723+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:54:29.783+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:54:29.782+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:54:29.877+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.240 seconds
[2024-01-04T20:54:59.988+0000] {processor.py:161} INFO - Started process (PID=7092) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:54:59.990+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:54:59.993+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:54:59.992+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:55:00.026+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:55:00.081+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:55:00.080+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:55:00.254+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:55:00.253+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:55:00.367+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.363 seconds
[2024-01-04T20:55:43.678+0000] {processor.py:161} INFO - Started process (PID=7147) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:55:43.680+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:55:43.690+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:55:43.682+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:55:43.800+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:55:44.518+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:55:44.511+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:55:44.582+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:55:44.582+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:55:44.631+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.962 seconds
[2024-01-04T20:56:23.528+0000] {processor.py:161} INFO - Started process (PID=7203) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:56:23.534+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:56:23.537+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:56:23.537+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:56:23.599+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:56:23.710+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:56:23.709+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:56:23.779+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:56:23.778+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:56:23.849+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.335 seconds
[2024-01-04T20:56:54.490+0000] {processor.py:161} INFO - Started process (PID=7260) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:56:54.494+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:56:54.500+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:56:54.499+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:56:54.590+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:56:54.659+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:56:54.658+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:56:54.725+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:56:54.724+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:56:54.776+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.294 seconds
[2024-01-04T20:57:25.437+0000] {processor.py:161} INFO - Started process (PID=7324) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:57:25.438+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:57:25.439+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:57:25.439+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:57:25.465+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:57:25.490+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:57:25.489+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:57:25.523+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:57:25.522+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:57:25.568+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.135 seconds
[2024-01-04T20:57:55.731+0000] {processor.py:161} INFO - Started process (PID=7383) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:57:55.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:57:55.734+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:57:55.734+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:57:55.782+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:57:55.810+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:57:55.810+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:57:55.845+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:57:55.845+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:57:55.947+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.219 seconds
[2024-01-04T20:58:26.054+0000] {processor.py:161} INFO - Started process (PID=7436) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:58:26.056+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:58:26.058+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:58:26.057+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:58:26.090+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:58:26.113+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:58:26.113+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:58:26.139+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:58:26.139+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:58:26.187+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.137 seconds
[2024-01-04T20:58:56.897+0000] {processor.py:161} INFO - Started process (PID=7495) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:58:56.899+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:58:56.901+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:58:56.900+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:58:56.938+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:58:56.964+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:58:56.964+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:58:56.997+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:58:56.996+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:58:57.037+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.144 seconds
[2024-01-04T20:59:27.141+0000] {processor.py:161} INFO - Started process (PID=7555) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:59:27.142+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:59:27.145+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:59:27.144+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:59:27.183+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:59:27.218+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:59:27.218+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:59:27.256+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:59:27.256+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:59:27.305+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.168 seconds
[2024-01-04T20:59:58.056+0000] {processor.py:161} INFO - Started process (PID=7620) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:59:58.057+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T20:59:58.059+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:59:58.059+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:59:58.105+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T20:59:58.164+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:59:58.163+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T20:59:58.260+0000] {logging_mixin.py:188} INFO - [2024-01-04T20:59:58.259+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T20:59:58.322+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.272 seconds
[2024-01-04T21:00:29.016+0000] {processor.py:161} INFO - Started process (PID=7679) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:00:29.018+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:00:29.020+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:00:29.019+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:00:29.055+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:00:29.083+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:00:29.083+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:00:29.117+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:00:29.116+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:00:29.161+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.152 seconds
[2024-01-04T21:00:59.475+0000] {processor.py:161} INFO - Started process (PID=7738) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:00:59.477+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:00:59.482+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:00:59.481+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:00:59.522+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:00:59.555+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:00:59.555+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:00:59.588+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:00:59.588+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:00:59.644+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.177 seconds
[2024-01-04T21:01:29.751+0000] {processor.py:161} INFO - Started process (PID=7797) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:01:29.753+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:01:29.754+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:01:29.754+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:01:29.787+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:01:29.816+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:01:29.816+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:01:29.853+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:01:29.853+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:01:29.894+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.147 seconds
[2024-01-04T21:02:00.466+0000] {processor.py:161} INFO - Started process (PID=7855) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:02:00.470+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:02:00.472+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:02:00.471+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:02:00.515+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:02:00.547+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:02:00.547+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:02:00.585+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:02:00.584+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:02:00.629+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.173 seconds
[2024-01-04T21:02:31.428+0000] {processor.py:161} INFO - Started process (PID=7914) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:02:31.430+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:02:31.431+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:02:31.431+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:02:31.456+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:02:31.493+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:02:31.493+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:02:31.529+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:02:31.529+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:02:31.566+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.142 seconds
[2024-01-04T21:03:01.968+0000] {processor.py:161} INFO - Started process (PID=7973) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:03:01.970+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:03:01.972+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:03:01.971+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:03:02.004+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:03:02.036+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:03:02.036+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:03:02.072+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:03:02.072+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:03:02.117+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.154 seconds
[2024-01-04T21:03:32.898+0000] {processor.py:161} INFO - Started process (PID=8032) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:03:32.900+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:03:32.902+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:03:32.902+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:03:32.930+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:03:32.962+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:03:32.962+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:03:33.013+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:03:33.012+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:03:33.076+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.181 seconds
[2024-01-04T21:04:03.405+0000] {processor.py:161} INFO - Started process (PID=8091) to work on /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:04:03.412+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dataset_dag.py for tasks to queue
[2024-01-04T21:04:03.441+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:04:03.413+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:04:03.554+0000] {processor.py:840} INFO - DAG(s) 'write_dag_dataset' retrieved from /opt/airflow/dags/dataset_dag.py
[2024-01-04T21:04:03.699+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:04:03.698+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2024-01-04T21:04:03.739+0000] {logging_mixin.py:188} INFO - [2024-01-04T21:04:03.738+0000] {dag.py:3820} INFO - Setting next_dagrun for write_dag_dataset to 2024-01-04T00:00:00+00:00, run_after=2024-01-05T00:00:00+00:00
[2024-01-04T21:04:04.020+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dataset_dag.py took 0.695 seconds
